types/johnny-five/index.d.ts

```diff
@@ -540,6 +548,22 @@ export declare class Motor {
     release(): void;
 }
 
+export declare class Motors {
+    constructor(option: Array<number> | MotorOption[]);
+
+    readonly isOn: boolean;
+
+    forward(speed: number): void;
+    fwd(speed: number): void;
+    reverse(speed: number): void;
+    rev(speed: number): void;
+    start(): void;
+    start(speed: number): void;
```

Comment:
Make this an optional argument and get rid of the above overload.

Reply:
Ok, I just copied from the Motor class, so I will also make it optional and delete the overload there